packages/deno/rollup.config.js

```diff
@@ -0,0 +1,51 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+import commonjs from '@rollup/plugin-commonjs';
+import typescript from '@rollup/plugin-typescript';
+import dts from 'rollup-plugin-dts';
+
+export default [
+  {
+    input: ['src/index.ts'],
+    output: {
+      dir: 'build',
+      sourcemap: true,
+      preserveModules: false,
+      strict: false,
+      freeze: false,
+      interop: 'auto',
+      format: 'esm',
+      banner: '/// <reference types="./index.d.ts" />',
+    },
+    plugins: [
+      nodeResolve({
+        extensions: ['.mjs', '.js', '.json', '.node', '.ts', '.tsx'],
+      }),
+      commonjs(),
+      replace({
+        preventAssignment: true,
+        values: {
+          __DEBUG_BUILD__: true,
+          __SENTRY_DEBUG__: true,
```

Comment:
h: We do not want to replace `__SENTRY_DEBUG__`. Users can do this if they want.

Reply:
Maybe we shouldn't be doing any replacements and treat it more like nodejs?

Reply:
Yeah we can do that actually. I was just wondering if there is any benefit to having slimmer bundles on serverside but it probably doesn't matter a lot and isn't worth the complexity we add.